tests/unit/amo/components/TestSearchFormSuggestion.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Suggestion from 'amo/components/SearchForm/Suggestion';
+
+
+describe(__filename, () => {
+  const shallowComponent = (props) => {
+    const allProps = {
+      search: '',
+      suggestion: '',
+      ...props,
+    };
+
+    return shallow(<Suggestion {...allProps} />);
+  };
+
+  it('renders itself', () => {
+    const props = {};
+    const root = shallowComponent(props);
+
+    expect(root.find('.Suggestion')).toHaveLength(1);
+  });
+
+  it('highlights parts of the suggestion that match the search', () => {
```

Comment:
"match the search term" or "match the search query" would make this spec a bit clearer.